server/data/data-jmap-postgres/src/main/java/org/apache/james/jmap/postgres/pushsubscription/PostgresPushSubscriptionDAO.java

```diff
@@ -65,7 +64,7 @@ public Mono<Void> save(Username username, PushSubscription pushSubscription) {
             .set(PushSubscriptionTable.USER, username.asString())
             .set(PushSubscriptionTable.DEVICE_CLIENT_ID, pushSubscription.deviceClientId())
             .set(PushSubscriptionTable.ID, pushSubscription.id().value())
-            .set(PushSubscriptionTable.EXPIRES, pushSubscription.expires().value().toLocalDateTime())
```

Comment:
At the time I checked this, it seems not only Postgres push subscription but the issue is at other places too (maybe 1 or 2 more places if I do not remember wrong).

Can you check again if any?

Reply:
yes, I checked, another uses LocalDateTime, but before saving to the database, they already converted by `.withZoneSameInstant(ZoneId.of("UTC"))`

Reply:
tldr: we can also quick fix this issue for push_subscriotion by quick convert `.withZoneSameInstant(ZoneId.of("UTC"))` 
But I am interested in how Postgres works with `timestamp with time zone`. So tried it for push_subscription

(cost for timestamp and timestamp with timezone is same, 8bit)